### ISSUE TITLE
Missing types from generated method parameters

### ISSUE THREAD

**Issue by ResoDev** [Milad-Akarie/injectable#9]
Using more strict lint rules makes injectable's singletons impossible to use. `_registerEagerSingletons` has dynamic parameters which produces `argument_type_not_assignable` error.

Generated code:
```dart
void _registerEagerSingletons(getIt, environment) {
	...
}
```

Should be:
```dart
void _registerEagerSingletons(GetIt getIt, String environment) {
	...
}
```

**Comment by Milad-Akarie** [comment]
@ResoDev noted.

**Comment by Milad-Akarie** [comment]
@ResoDev  fixed in v0.2.2
Thanks for pointing that out.
